content/blog/the-8-part-guide-to-better-code-review-checklists/index.md

```diff
@@ -0,0 +1,124 @@
+
+Although software development isn't exactly a life-or-death endeavor, we can apply the same lessons by using the power of checklists to priorize effectively when collaborating.
+
+## What is a code review?
+
+When a software developer on a team wants to contribute code they've worked on back into the main project, they need to do a 'pull request' or 'merge request'. It's the main way developers contribute to software projects, but [sometimes getting pull requests approved can be agonizingly slow](https://featurepeek.com/blog/why-are-so-many-orgs-slow-to-approve-pull-requests/).
+
+Before this code can be added to the project, other team members will do a code review. This means checking for bugs, issues and suggesting improvements.
+
+## Why make a code review checklist?
+
+Generally speaking, code reviews are great. They radically improve code quality, increase developer productivity and prevent bugs from ever reaching customers.
+
+But without a good process, code reviews can be, in a word...painful.
+
+Code review checklists help ensure productive code reviews. A code review checklist prevents simple mistakes, verifies work has been done and helps improve developer performance.
+
+# 1. Pull Request Etiquette ✅
+
+Start with the basics. Is the pull request you are looking at *actually* ready for review? To determine this, here are some key principles that should apply to all code reviews:
+
+* **Frequent** — To make pull requests more manageable, submit them little and often. If a pull request covers multiple days with no good reason, it should be split into smaller chunks.
+* **Bitesize** — If a pull request is too large, it's difficult to understand. It should be broken down as much as possible.
+* **Well documented** — Does the pull request have enough comments to make it easy to review?
+* **Follows the Single Responsibility Principle** — Code should only ever solve one problem. If a piece of code affects a lot of the rest of the code in the project, it's difficult to review.
+
+If these conditions are not met, return the code to the contributor to be improved or separated into different pull requests.
+
+This might seem initially harsh or counter-productive, but over several weeks your team's outputs and productivity will improve.
+
+# 2. Styling 🎨
+
+Set some basic styling rules for your team and ensure these are followed. Consistent code styling is essential to ensure future developers can understand code easily and work productively.
+
+Some basic rules:
+* Are variable names sensible and consistently capitalized?
+* Are there sufficiently descriptive comments throughout the code as required?
+* Are formatting preferences followed? For example, tabs or spaces, curly brace on the same line or new line, 80 character width or 120 character width?
+
+To save time, you could check these rules are followed with an automatic code linter.
```

Comment:
[Prettier](https://prettier.io) is a good example to link to for JavaScript projects.